pkg/minikube/bootstrapper/certs.go

```diff
@@ -360,9 +362,16 @@ func collectCACerts() (map[string]string, error) {
 
 // getSubjectHash calculates Certificate Subject Hash for creating certificate symlinks
 func getSubjectHash(cr command.Runner, filePath string) (string, error) {
+	lrr, err := cr.RunCmd(exec.Command("ls", "-la", filePath))
+	if err != nil {
+		return "", err
```

Comment:
Wrap the error with something helpful.

Reply:
I don't think there is anything extra we can add - the error already includes the command-line:

getSubjectHash: ls -la <path>: returned exit code 1